olp-cpp-sdk-core/tests/cache/DefaultCacheImplTest.cpp

```diff
@@ -197,8 +203,9 @@ TEST_F(DefaultCacheImplTest, LruCachePut) {
 
     cache.Open();
     cache.Clear();
-    cache.Put(key, data_string, [=]() { return data_string; },
-              (std::numeric_limits<time_t>::max)());
+    cache.Put(
+        key, data_string, [=]() { return data_string; },
+        (std::numeric_limits<time_t>::max)());
```

Comment:
Please make sure you are using clang-format v6 so that we do not have this reformatting issues.

Reply:
I have tried clang-format-5 but it does not accept some options, so I'm using clang-format-10
What clang-format should I use?

Reply:
We are all using `clang-format version 6.0.0-1ubuntu2 (tags/RELEASE_600/final)`